LICENCE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (C) 2013 Crown Copyright (Government Digital Service)
```

Comment:
The range of years is used when significant changes are made in subsequent years. I don't know this repo well enough to know whether the core functionality dates from 2013 or whether there are subsequent changes that could qualify for copyright. What do you think?

More info here: http://www.copyrightservice.co.uk/copyright/p03_copyright_notices